docs/README.md

```diff
@@ -42,44 +54,99 @@ source ~/ChildProjectVenv/bin/activate
 pip install git+https://github.com/LAAC-LSCP/ChildRecordsData.git
 ```
 
-2. Install git-annex
+### 2. Install git-annex
 
 The package also requires git-annex in order to retrieve the datasets. It can be installed via `apt install git-annex` or `brew install git-annex`. It is encouraged to use a recent version of git-annex (8.2+).
 
-## Installing datasets
+## Getting some data
+
+You can either have some data of your own that you would like to use the package on, or you may know of some datasets that are already in this format that you'd like to reuse.
+
+It may be easier to start with an extant dataset. Here is the list that we know exists. Please note that the large majority of these data are NOT public, and thus if you cannot retrieve them, this means you need to get in touch with the data managers.
+
+### Extant datasets
+
+Instructions to download extant datasets can be found [here](http://laac-lscp.github.io/ChildRecordsData/REUSE.html).
+
+#### Public data sets (TODO)
+
+We have prepared a public data set for testing purposes which is based on the [VanDam Public Daylong HomeBank Corpus](https://homebank.talkbank.org/access/Public/VanDam-Daylong.html); VanDam, Mark (2018). VanDam Public Daylong HomeBank Corpus. doi:10.21415/T5388S.
+
+
 
-The list of available datasets with instructions to download them can be found [here](http://laac-lscp.github.io/ChildRecordsData/PROJECTS.html).
+#### From the [LAAC team](https://lscp.dec.ens.fr/en/research/teams-lscp/language-acquisition-across-cultures)
 
-## Working with the data
+
+| Name | Authors | Location | Recordings | Audio length (hours) | Status |
```

Comment:
This is dynamic (i.e. filled with Jinja templating engine at generation time)
But README.md is static (and as the entry point for the docs on github, should remain as such in my opinion)
So everything that is dynamic should not be on the README (or just filled statically)

Reply:
roger! i'll move it then